packages/apps/fortune/recording-oracle/src/modules/job/job.service.ts

```diff
@@ -165,25 +227,21 @@ export class JobService {
 
       return 'The requested job is completed.';
     }
-
-    return 'Solution is recorded.';
-  }
-
-  private async uploadJobSolutions(
-    jobSolutions: any[],
-    bucket: string,
-  ): Promise<SaveSoulutionsDto> {
-    const uploadedFiles: UploadFile[] = await this.storageClient.uploadFiles(
-      jobSolutions,
-      bucket,
-    );
-
-    if (!uploadedFiles[0]) {
-      this.logger.log(ErrorBucket.UnableSaveFile, JobService.name);
-      throw new BadGatewayException(ErrorBucket.UnableSaveFile);
+    if (errorSolutions.length) {
+      const exchangeOracleURL = (await kvstoreClient.get(
+        exchangeJobSolutionsFile.exchangeAddress,
+        'webhook_url',
+      )) as string;
+      for (const solution of errorSolutions) {
+        await this.sendWebhook(exchangeOracleURL + '/invalid-solution', {
+          chainId: jobSolution.chainId,
+          escrowAddress: jobSolution.escrowAddress,
+          solution,
+        });
+      }
     }
 
-    return uploadedFiles[0];
+    return 'Solution are recorded.';
   }
 
   public async sendWebhook(
```

Comment:
Try to move this request to a utils class and set webhookData to any or similar since depending on the case the data passed will be different